src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -276,7 +276,7 @@ public void testDropChannel() throws Exception {
   @Test
   public void testParameterOverrides() throws Exception {
     Map<String, Object> parameterMap = new HashMap<>();
-    parameterMap.put(ParameterProvider.MAX_CLIENT_LAG, "3 sec");
```

Comment:
why this is changed?

Reply:
The ParameterProvider does not support "sec" ([ref](https://github.com/snowflakedb/snowflake-ingest-java/blob/f69259a81d849948dabf72af88d48f7e1d94e7f6/src/main/java/net/snowflake/ingest/utils/ParameterProvider.java#L225C10-L228C19)). The old code somehow ignore the exception thrown in thread.